### PR TITLE
Add OpenGraph metadata

### PR DESCRIPTION
**Pull request by jc4p**
I believe it also works on Twitter so we don't need to also add Twitter
card metadata.

Also updates the first post to have new metadata + image.

Facebook requires a fully qualified URL for the `og:url` parameter, so I had to also add the site's host to the config. Hopefully that doesn't break the Github Pages setup.

### CODE REVIEW

File: _includes/head.html [hunk 1]
```diff
@@ -5,21 +5,21 @@
...
-<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
-<script src="js/scripts.js"></script>
-<script>
-  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+  <link rel="stylesheet" href="https://unpkg.com/tachyons@4.7.0/css/tachyons.min.css"/>
+  <link rel="stylesheet" href="{{ site.baseurl }}/css/main.css">
+  {% if extra_metadata %}{{ extra_metadata }}{% endif %}
```
Comment (jc4p) [inline _includes/head.html:12]: this is the only change, the rest is indentation

**Comment by jyng** [comment]
Lets Get This Merged

**Review by jyng — COMMENTED** [review]
LGTM

**Comment by hellophien** [comment]
Who are you talking to?

**Comment by jyng** [comment]
dude

**Pull request merged**